internal/metamorph/server.go

```diff
@@ -165,6 +165,13 @@ func ValidateCallbackURL(callbackURL string) error {
 		return nil
 	}
 
+	rejectedCallbackUrlSubstrings := viper.GetStringSlice("metamorph.rejectCallbackContaining")
```

Comment:
I think this validation should happen in `api` service.

We already have here one validation: https://github.com/bitcoin-sv/arc/blob/277ef146f0210d093daeb33438050bdf082657d8/pkg/api/handler/default.go#L338